cms/djangoapps/contentstore/features/transcripts.feature

```diff
@@ -1,84 +0,0 @@
-@shard_1 @requires_stub_youtube
```

Comment:
Isn't this video stuff, not assets page stuff?

Reply:
All of the lettuce clean up (ie. features/upload, this, and features/video) are old tests that exist in bokchoy already. The video test below is also testing by uploading a video with the old studio assets page which isn't the workflow anymore either, they should be using the video uploads page for videos, so it wasn't even testing the correct workflow. The video and transcripts files were kind of the ripple effect of lettuce failures when I removed the old assets page from the system.

Reply:
ooh I see. Well, that is good that we are removing old unneeded code :+1: